### DISCUSSION TITLE
Is there a way to copy or export the current log output

### DISCUSSION THREAD

**Discussion by zizther** [spatie/ray#558] [Q&A]
I am using the Ray app on macOS.
Is there a way to copy or export all the current log output?

When I select all the log outputs and copy these it only pastes a certain number of log outputs, not sure if this is a macOS limitation or a bug.

If there is no way to copy or export, this would be a nice feature to add in.

**Comment by AdrianMrn** [comment]
Because of the sheer amount of types of logs, and what their actual underlying content is (Symfony dumps for example are just plain html), we've decided to leave this feature out for now. We've enabled this in -in our opinion- the most important places, like sql queries.

**Reply by zizther** [reply]
Thanks for the update.

**Reply by redhedded1** [reply]
I'd like to have a way to get the output out of ray after a test run, into a text editor for review. Is there a work around?
